docs/MockserverAPI.md

```diff
@@ -18,13 +18,62 @@ You just have to return an array of objects that will be used as the initial dat
 - `getInitialDataSet?: (contextId: string) => object[];`
   - `contextId` represents the tenant id for the current request
 
+### fetchEntries
```

Comment:
fetchEntries override only makes sense in parametrized entity in my opinion, in most case you can do what you need in getInitialDataSet, that's why it's not documented currently and why i'd rather keep it like that so not to confuse consumers